jetty-websocket/websocket-client/src/main/java/org/eclipse/jetty/websocket/client/ClientUpgradeRequest.java

```diff
@@ -179,6 +181,27 @@ public void setRequestURI(URI uri)
         }
     }
 
+    /**
+     * @param timeout the total timeout for the request/response conversation of the WebSocket handshake;
+     * use zero or a negative value to disable the timeout
+     * @param unit the timeout unit
+     * @return this request object
+     */
+    public ClientUpgradeRequest timeout(long timeout, TimeUnit unit)
```

Comment:
To be consistent with other methods of this class, this should be `void setTimeout(long, TimeUnit).